modules/NuGet.Tasks/module.targets

```diff
@@ -0,0 +1,41 @@
+<Project>
+
+  <PropertyGroup>
+    <RestoreDependsOn>ApplyNuGetPolicies;$(RestoreDependsOn)</RestoreDependsOn>
+    <ApplyNuGetPoliciesDependsOn>
+      $(ApplyNuGetPoliciesDependsOn);
+      ResolveSolutions;
+      $(PrepareDependsOn);
+    </ApplyNuGetPoliciesDependsOn>
+  </PropertyGroup>
+
+  <ItemGroup Condition="@(PackageLineup->Count()) != 0 AND '$(RestrictVersionOnPackageReference)' != 'false' ">
+    <!-- Automatically warn about package reference with versions this when lineups are used. -->
```

Comment:
grammar error. it said something different at first and i failed to delete "this"